src/hooks/useProofAddressesAvailableToCreate.ts

```diff
@@ -1,47 +1,44 @@
-import {
-  GoerliContractsStore,
-  MainnetContractsStore,
-} from 'stores/ContractsStore'
 import { useSnapshot } from 'valtio'
+import ContractsStore from 'stores/ContractsStore'
 import Network from 'models/Network'
 import ProofStore from 'stores/ProofStore'
 import SealCredStore from 'stores/SealCredStore'
-import networkPick from 'helpers/networkPick'
+import useAllContractsOwned from 'hooks/useAllContractsOwned'
 import useContractsOwned from 'hooks/useContractsOwned'
 
 export default function (network?: Network) {
-  const { goerliERC721ProofsCompleted, mainnetERC721ProofsCompleted } =
-    useSnapshot(ProofStore)
+  const { eRC721ProofsCompleted } = useSnapshot(ProofStore)
+  const { derivativeContracts = [] } = useSnapshot(SealCredStore)
+
   let contractsOwned: readonly string[]
   switch (network) {
     case Network.Mainnet:
```

Comment:
If `Mainnet` has empty case, maybe we should remove this line at all?

Reply:
Then the case won't work for the mainnet, right?)